hack/gosec.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+# Example:  ./hack/gosec.sh
+set -x
+
+if [ "$IS_CONTAINER" != "" ]; then
+  go get github.com/securego/gosec/cmd/gosec
```

Comment:
Our [only precedent for a local install][1] in a `hack/` script is gated on command existence and uses `go get -u`, both of which seem like good ideas to me.

[1]: https://github.com/openshift/installer/blob/d89a59f68e2d7786f585242925f040fc645c9a54/hack/verify-vendor.sh#L4

Reply:
ack, will change this.

Reply:
done.